Git Streak Tracker/ContentView.swift

```diff
@@ -38,45 +38,47 @@ struct ContentView: View {
             .animation(.easeInOut, value: viewStore.selectedTab) // 2
             
             if !userStore.username.isEmpty {
-                HStack {
-                    
-                    Spacer()
-                    
-                    FontIcon.button(
-                        .materialIcon(code: .person),
-                        action: {
-                            viewStore.switchTab(tab:0)
-                        },
-                        padding: 0,
-                        fontsize: 45,
-                        color: viewStore.selectedTab == 0 ? ColorPallete.highlightGreen : ColorPallete.midWhite
-                    )
-                    .shadow(color: viewStore.selectedTab == 0 ? ColorPallete.highlightGreen : .clear, radius: 4, x: 0, y: 0)
-                    
-                    Spacer()
-                    Spacer()
-                    
-                    FontIcon.button(
-                        .ionicon(code: .ios_settings),
-                        action: {
-                            viewStore.switchTab(tab:1)
-                        },
-                        padding: 0,
-                        fontsize: 45,
-                        color: viewStore.selectedTab == 1 ? ColorPallete.highlightGreen : ColorPallete.midWhite
-                    )
-                    .shadow(color: viewStore.selectedTab == 1 ? ColorPallete.highlightGreen : .clear, radius: 4, x: 0, y: 0)
-                    
-                    Spacer()
-                    // When we wanna animate this all pretty like the designs, watch this: https://www.youtube.com/watch?v=lzmKrJCuxwM&t=221s&ab_channel=Kavsoft
-                    
+                GeometryReader { geometry in
```

Comment:
This section just got wrapped in `GeometryReader`. I'll point out the one other line that was changed.